chia/data_layer/data_store.py

```diff
@@ -655,3 +749,55 @@ async def get_proof_of_inclusion_by_key(
         async with self.db_wrapper.locked_transaction(lock=lock):
             node = await self.get_node_by_key(key=key, tree_id=tree_id, lock=False)
             return await self.get_proof_of_inclusion_by_hash(node_hash=node.hash, tree_id=tree_id, lock=False)
+
+    async def get_left_to_right_ordering(
```

Comment:
What is this doing?

Reply:
produces the left to right tree traversal starting from `node_hash`. this way client can ask server for nodes exactly from the place it downloaded last. also server isn't limited to `node_hash` subtree - if subtree is out of nodes, but query size still fits more nodes - we'll go up and right to find a new subtree.